models/DataObject/ClassDefinition.php

```diff
@@ -408,6 +408,7 @@ public function generateClassFiles($generateDefinitionFile = true)
         $cd .= 'use Pimcore\Model\DataObject\PreGetValueHookInterface;';
         $cd .= "\n\n";
         $cd .= "/**\n";
+        $cd .= '* @method static \\Pimcore\\Model\\DataObject\\'.ucfirst($this->getName()).'\Listing getList()'."\n";
```

Comment:
Is it `getList()` or `getListing()`?

Reply:
getList() https://github.com/pimcore/pimcore/blob/023f10bf34e9c462eeb03ae782740570f7168f01/models/DataObject/AbstractObject.php#L357